pennylane/transforms/draw.py

```diff
@@ -149,3 +149,181 @@ def wrapper(*args, **kwargs):
         )
 
     return wrapper
+
+
+def draw_mpl(qnode, wire_order=None, show_all_wires=False, decimals=None, **kwargs):
+    """Draw a qnode with matplotlib
+
+    Args:
+        qnode (.QNode): the input QNode that is to be drawn.
+
+    Keyword Args:
+        wire_order (Sequence[Any])
+        show_all_wires (bool)
+        wire_options (dict)
+        label_options (dict)
+
+    Returns:
+        A function that has the same argument signature as ``qnode``. When called,
+        the function will draw the QNode as a tuple of (``matplotlib.figure.Figure``,
+        ``matplotlib.axes._axes.Axes``)
+
+    **Example**:
+
+    .. code-block:: python
+
+        dev = qml.device('lightning.qubit', wires=(0,1,2,3))
+
+        @qml.qnode(dev)
+        def circuit(x, z):
+            qml.QFT(wires=(0,1,2,3))
+            qml.Toffoli(wires=(0,1,2))
+            qml.CSWAP(wires=(0,2,3))
+            qml.RX(x, wires=0)
+            qml.CRZ(z, wires=(3,0))
+            return qml.expval(qml.PauliZ(0))
+
+
+        fig, ax = draw_mpl(circuit)(1.2345,1.2345)
+
+    .. figure:: ../../_static/draw_mpl_qnode/main_example.png
+            :align: center
+            :width: 60%
+            :target: javascript:void(0);
+
+    .. UsageDetails::
+
+    **Decimals:**

```

Comment:
don't forget to indent everything below `UsageDetails`! Otherwise Sphinx won't consider it part of `UsageDetails`: